src/Symfony/Component/Finder/Comparator/Comparator.php

```diff
@@ -18,8 +18,8 @@
  */
 class Comparator
 {
-    private $target;
-    private $operator = '==';
+    private ?string $target = null;
```

Comment:
the type of the getter probably needs to be updated (or this should default to the empty string).

Reply:
Calling the getter before initialization does not really make sense. The two child classes of `Comparator` do initialize the property via the constructor. Let's skip the initialization.

Reply:
Let's make that class a bit more robust: #42137